### ISSUE TITLE
Alchemy issue with Vacuous

### ISSUE THREAD

**Issue by Nighthawk08** [Azanor/thaumcraft#1682]
I know there was a previous issue with vacuous and alchemy. However, I have version 4.2.3.4 and it is will an issue. 

For instance, I put four stacks of bowls into the Alchemical Furnace, and only received 7 vacuous. This is the same with Chests, though I dont have exact number values. just that both melt down entirely within seconds.

**Comment by dmportella** [comment]
I can confirm this issue, it's been seen on other mod pack like realm of miamite I saw even Sonia experience this. For chests for example it consumes it strait away an entire stack but you only get 1 aspect from it.

**Comment by Azanor** [comment]
Yeah, its a known and annoying issue. I usually has to do with other mods adding recipes that bend/breaks my aspect calc algorithm. I am looking into this.
